simuleval/data/dataloader/dataloader.py

```diff
@@ -52,15 +56,28 @@ def get_source(self, index: int) -> Any:
     def get_target(self, index: int) -> Any:
         return self.preprocess_target(self.target_list[index])
 
+    def get_tgt_lang(self, index: int) -> Any:
+        if self.tgt_lang_list is not None and index < len(self.tgt_lang_list):
+            return self.preprocess_tgt_lang(self.tgt_lang_list[index])
```

Comment:
Don't thin we need a preprocessor for tgt_lang attribute, preprocessor makes more sense for source and targets which can potentially be sound files.